src/query.pegjs

```diff
@@ -2,6 +2,7 @@
   //TODO: disabled presets (e.g. mongo 2.2/2.6/3.0)
   //TODO: determine dependencies automatically
   if (!Array.isArray(options.disabledOperators)) options.disabledOperators = [];
+  if (options.caseInsensitiveOperators === undefined) options.caseInsensitiveOperators = false;
```

Comment:
Well that was an education.  undefined is a global.  If I'd read that before I must have forgotten about it.  Thankfully I'd never do that and linting and the way commonjs loads modules this isn't really unsafe here.